config/initializers/invisible_captcha.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+InvisibleCaptcha.setup do |config|
+  # Disable timestamp check for test environment
+  config.timestamp_enabled = !Rails.env.test?
+end
```

Comment:
Could this be in the spec helper instead?

Reply:
Done here 048619d660cc30112449d0a353acd61597c49657